glide.yaml

```diff
@@ -44,34 +41,28 @@ import:
   version: ^1.10
 - package: golang.org/x/net
   version: master
-  # explicitly specifying this because glide is having issues with golang.org repos
   repo: https://github.com/golang/net
   subpackages:
   - context
 - package: google.golang.org/grpc
   version: ^1.12.0
   repo: https://github.com/grpc/grpc-go
 - package: golang.org/x/sys
-  # explicitly specifying this because glide is having issues with golang.org repos
-  # this is just a dependency of golang.org/x/net and if this problem is fixed
-  # then we no longer need to have this dependency in glide.yaml
   repo: https://github.com/golang/sys
 - package: golang.org/x/tools
-  # explicitly specifying this because glide is having issues with golang.org repos
-  # this is just a dependency of golang.org/x/net and if this problem is fixed
-  # then we no longer need to have this dependency in glide.yaml imports
-  # we should then move it to testImports
   repo: https://github.com/golang/tools
 - package: gopkg.in/yaml.v2
 - package: go.uber.org/multierr
   version: '>= 0.1, < 2.0'
 - package: github.com/golang/mock
   version: ^1
+- package: github.com/gogo/googleapis
```

Comment:
Let's revisit this. Ideally we wouldn't need to depend on this package.

Reply:
From https://jbrandhorst.com/post/grpc-errors/, this appears to be non-actionable for now. I'll investigate our dependency on `gogo/protobuf` in a separate change.